src/util.js

```diff
@@ -869,3 +876,23 @@ export function focusTextArea(element) {
 
     return textAreaEl;
 }
+
+/**
+ * Generates a blank mobile annotation dialog
+ *
+ * @return {HTMLElement} Blank mobile annotation dialog
+ */
+export function regenerateMobileDialog() {
```

Comment:
Could we combine this with resetMobileDialog? Based on the naming it should do  the same thing?

Reply:
I renamed annotator.resetMobileDialog() and changed it to call the util method. I'm going to spend a little time refactoring the shared dialog logic only in the annotator. I think it might require a little more refactoring than this bug fix should contain. 

annotator.resetMobileDialog() (now called annotator.resetSharedDialog()) clears out the contents of the shared mobile dialog 

util.regenerateMobileDialog() (now called util.generateMobileDialogEl()) generates a blank dialog to replace the one stored in each annotation dialog. To move away from this will require some refactoring. I can tackle that in a separate commit.